.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch: { }
+
+jobs:
+  tests:
+    name: Tests
+    uses: ./.github/workflows/test.yml
+  deploy-snapshots:
```

Comment:
:question: Do you think we might need [concurrency](https://docs.github.com/en/actions/deployment/about-deployments/deploying-with-github-actions#using-concurrency) here, to avoid concurrent deployments of the artifacts?

Reply:
Yes, see line 18 👍

Reply:
Haha, how did I not notice it :see_no_evil: